ql/src/java/org/apache/hadoop/hive/ql/ddl/table/create/CreateTableDesc.java

```diff
@@ -958,6 +940,31 @@ public Table toTable(HiveConf conf) throws HiveException {
     return tbl;
   }
 
+  public static void setColumnsAndStorePartitionTransformSpec(
```

Comment:
Consider moving the method in `DDLUtils` or `AlterTableUtils`.

Reply:
Consider including `table` somewhere in the method name to better reflect its purpose.